cmd/resource.go

```diff
@@ -0,0 +1,113 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/goto/salt/printer"
+	"github.com/goto/shield/pkg/file"
+	shieldv1beta1 "github.com/goto/shield/proto/v1beta1"
+	cli "github.com/spf13/cobra"
+)
+
+func ResourceCommand(cliConfig *Config) *cli.Command {
+	cmd := &cli.Command{
+		Use:     "resource",
+		Aliases: []string{"resources"},
+		Short:   "Manage resources",
+		Long: heredoc.Doc(`
+			Work with resources.
+		`),
+		Example: heredoc.Doc(`
+			$ shield resource config upsert
+		`),
+		Annotations: map[string]string{
+			"group":  "core",
+			"client": "true",
+		},
+	}
+
+	cmd.AddCommand(resourceConfigCommand(cliConfig))
+
+	bindFlagsFromClientConfig(cmd)
+
+	return cmd
+}
+
+func resourceConfigCommand(cliConfig *Config) *cli.Command {
+	cmd := &cli.Command{
+		Use:   "config",
+		Short: "Manage resources config",
+		Long: heredoc.Doc(`
+			Work with resources config.
+		`),
+		Example: heredoc.Doc(`
+			$ shield resource config upsert
```

Comment:
probably upload name is better here
```suggestion
			$ shield resource config upload
```